qiskit/circuit/library/standard_gates/dcx.py

```diff
@@ -48,14 +48,9 @@ class DCXGate(SingletonGate):
             \end{pmatrix}
     """
 
-    def __init__(self, label=None, duration=None, unit=None, _condition=None):
+    def __init__(self, label=None, *, duration=None, unit="dt"):
```

Comment:
Oh, this actually was a bit of a mistake that I changed what you originally had - I was just writing it on autopilot, mechanically and manually undoing the mistake I made where I'd removed _all_ the `label` and `duration` arguments.  But yeah, since we've got the opportunity, let's just do it.